github-dark.css

```diff
@@ -8726,7 +8725,8 @@
   .site-header-nav .dropdown-menu a.active,
   .site-header-nav .dropdown-menu a .octicon-check,
   .marketplace-product-callout .branch-action-item-icon,
-  .user-status-org-button span.text-bold, .octicon-fold {
+  .user-status-org-button span.text-bold, .octicon-fold,
+  .jump-to-suggestions-path {
```

Comment:
Either this or something else, but this looks more balanced to me between highlight and normal.